api-samples/devtools/chrome-query/chrome-query.md

```diff
@@ -0,0 +1,15 @@
+# devtools
+
+This sample demonstrates using the devtools API to add a sidebar that displays the jQuery data associated with the selected DOM element.
```

Comment:
Is it still worth demoing anything related to JQuery?

Reply:
Long term I think a sample that doesn't use jQuery would be nice, but jQuery still exists, and so I don't think we necessarily need to change this given that our main goal is just having something that works in MV3. This feels like a useful sample for developers migrating even if the context is a bit less common these days.